bin/language.txt

```diff
@@ -817,7 +817,7 @@ Monarchy
 Republic
 Fundamentalism
 Communism
-Democracy
+Capitalism
```

Comment:
it was my test change - I was trying how github works. My real "push" is for delphi ( i think it is your "default" branch)

name Democracy is define in Language and once in Help for Fanatic feature description.

I understand that you want to only rewrite c-evo code and include maybe UI changes.
*(its realism&historic accuracy: Democracy was in ancient Greece, Rome. By the age when Statue of liberty was introduce the Capitalism appear (and socialism and communism)).*